### ISSUE TITLE
Support plain files

### ISSUE THREAD

**Issue by jtpio** [QuantStack/jupyter-suggestions#10]
Currently `ISuggestionsModel` expects a `NotebookPanel`:

https://github.com/QuantStack/jupyterlab-suggestions/blob/26b00123d50a846cf200c9c49488bd1c2279a79d/packages/base/src/types.ts#L28-L31

Since suggestions are currently displayed in the right sidebar, it could be interesting to also consider supporting suggestions for plain files, such as Python or Markdown files.

Although this may add some challenges as to where the suggestions should be saved, as there would be no notebook metadata for such files: https://github.com/QuantStack/jupyterlab-suggestions/issues/1

**Comment by trungleduc** [comment]
yeah, i will get back to the plain files later after the integration with the jupyter-collaboration backend

**Comment by trungleduc** [comment]
we probably need to handle the plain files differently

**Comment by jtpio** [comment]
Sounds good :+1: 

Also it's fine to completely break and change interfaces as needed now since we are still in the early iterations.
